### PR TITLE
Fix name in ENV variable in podspec action

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @KrauseFx</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by taquitos** [comment]
@KrauseFx is this a new variable or will people need to update?

File: fastlane/lib/fastlane/actions/version_get_podspec.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def self.available_options
                                          UI.user_error!("Please pass a path to the `version_get_podspec` action") if value.length == 0
                                        end),
           FastlaneCore::ConfigItem.new(key: :require_variable_prefix,
-                                       env_name: "FL_VERSION_BUMP_PODSPEC_VERSSION_REQIURE_VARIABLE_PREFIX",
+                                       env_name: "FL_VERSION_PODSPEC_VERSION_REQIURE_VARIABLE_PREFIX",
```
Comment (revolter) [inline fastlane/lib/fastlane/actions/version_get_podspec.rb]: `REQUIRE` instead of `REQIURE`

**Review by revolter — CHANGES_REQUESTED** [review]
Same

**Comment by revolter** [comment]
@taquitos, It's from #11723.

File: fastlane/lib/fastlane/actions/version_get_podspec.rb (hunk 1 above)
Comment (revolter) [reply fastlane/lib/fastlane/actions/version_get_podspec.rb]: Also, `BUMP` is missing!

**Comment by taquitos** [comment]
@revolter I just updated the typo 😄

**Review by revolter — APPROVED** [review]

**Pull request merged**